pkg/pgmodel/ingestor/ingestor.go

```diff
@@ -39,12 +40,18 @@ func NewPgxIngestor(conn pgxconn.PgxConn, cache cache.MetricCache, sCache cache.
 	return &DBIngestor{dispatcher: dispatcher, sCache: sCache, parser: parser}, nil
 }
 
+// IngestorTestConfig is a config for NewPgxIngestorForTests.
+type IngestorTestConfig struct {
+	Conn             pgxconn.PgxConn
+	DecompressChunks bool
```

Comment:
This should be IgnoreCompressedSamples bool instead so that the default is the zero-value(false). That way we can avoid DecompressChunks: true all over the place.

Reply:
I had thought about that, but felt that decompressChunks is easier to read and suits our case.